1.2 only: add a note to manually move pods if necessary (#1745)
Cray-HPE/docs-csm: operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md

Only follow the steps in the section for the node type that is being rebuilt:

* Worker node
  

>   1. Make sure not all pods of `ingressgateway-hmn, spire-server` are running on the same worker nodes. If all pods of above deployments are placed onto one worker node, then use `/opt/cray/platform-utils/move_pod.sh` to move at least one pod to a different worker noede.

Reviewer: ```suggestion
    1. Make sure that not all pods of `ingressgateway-hmn` or `spire-server` are running on the same worker node.
  
          For either of those two deployments, if all pods are running on a single worker node, then use the
          `/opt/cray/platform-utils/move_pod.sh` script to move at least one pod to a different worker node.
```

> 

Reviewer: ```suggestion

    1. Rebuild the worker node.

```

>   ```bash

Reviewer: ```suggestion
        ```bash
```

>   ncn-m001# /usr/share/doc/csm/upgrade/1.2/scripts/rebuild/ncn-rebuild-worker-nodes.sh ncn-w001

Reviewer: The line after this one also needs to be indenter further
```suggestion
        ncn-m001# /usr/share/doc/csm/upgrade/1.2/scripts/rebuild/ncn-rebuild-worker-nodes.sh ncn-w001
```
